doc/documentation.rst

```diff
@@ -1,73 +0,0 @@
-.. _documentation:
-
-=============
-Documentation
-=============
-
-:Release: |version|
-:Date: |today|
-
-Previous versions: `1.3.0 <http://nipype.readthedocs.io/en/1.3.0/>`_ `1.2.3 <http://nipype.readthedocs.io/en/1.2.3/>`_
```

Comment:
Is this something we still want to include somewhere? Possibly the new `interfaces.rst`?

Reply:
I have no problem with this, but it is true that having it adds some maintenance.

Reply:
I just put it back in interfaces.rst.